en/mkdocs.yml

```diff
@@ -457,7 +457,7 @@ nav:
                 - Mitigating Cross Site Request Forgery Attacks: Administer/ProductSecurity/UsingSymmetricEncryption/mitigating-cross-site-request-forgery-attacks.md
                 - Mitigating Cross Site Scripting Attacks: Administer/ProductSecurity/UsingSymmetricEncryption/mitigating-cross-site-scripting-attacks.md
             - Enabling HostName Verification: Administer/ProductSecurity/enabling-hostname-verification.md
-            - Configuring TLS Termination: Administer/ProductSecurity/configuring-tls-termination.md
+            # - Configuring TLS Termination: Administer/ProductSecurity/configuring-tls-termination.md
```

Comment:
@RAVEENSR , did u comment out this link in the YAML by accident? Just wondering because you have renamed this page en/docs/Administer/ProductSecurity/configuring-tls-termination.md

Reply:
Rationale behind this move is described in the PR description

Reply:
Great. Sorry, I missed that information.